### DISCUSSION TITLE
Add "Trigger Now" to Webhook flow

### DISCUSSION THREAD

**Discussion by lukas-schaetzle** [directus/directus#15481] [Feature Requests]
There's currently no simple option to manually trigger a flow which is not linked to a certain collection from inside the app. Therefore, I suggest adding a "trigger now" option to flows with a webhook trigger. Selecting that new option would simply call the webhook address.

![grafik](https://user-images.githubusercontent.com/45203603/189131138-1005d1da-cd19-43d7-865c-b50fa517944d.png)

**Comment by benhaynes** [comment]
This would need to be controlled by the trigger, since some require a payload and some don't. Only the latter could be triggered manually... Unless we also included a modal to enter a payload or certain parameters.

**Reply by rijkvanzanten** [reply]
Yup! The only way we can make this work reliably is by forcing the user to mock the trigger a nd accountability payload data that's normally used. Otherwise, your flow will fail if you're relying on any of the `$trigger` data f.e.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
